plugin/plugins/file/service/service.go

```diff
@@ -262,12 +314,40 @@ func (fss *StreamingService) Stream(wg *sync.WaitGroup) error {
 	return nil
 }
 
-// Close satisfies the io.Closer interface, which satisfies the baseapp.StreamingService interface
-func (fss *StreamingService) Close() error {
+// Close satisfies the io.Closer interface
+func (fss *FileStreamingService) Close() error {
 	close(fss.quitChan)
 	return nil
 }
 
+// ListenSuccess returns a chan that is used to acknowledge successful receipt of messages by the external service
+// after some configurable delay, `false` is sent to this channel from the service to signify failure of receipt
+func (fss *FileStreamingService) ListenSuccess() <-chan bool {
```

Comment:
Moreover, I think that `ackChan` should be a parameter of this function, rather being part of the struct. Returning a private value / chan is smells badly.